genon/preprocessor/converters/xlsx_processor.py

```
@@ -505,3 +505,95 @@ def build_tabular_vectors(
+) -> dict:
+    """xlsx/csv → {"data":[{"sheet_name","title","data_rows":[{col:val}]}]} (중립 표현).
+
+    표 감지(멀티헤더 자동 + 1시트 복수표)는 load_tables 에 위임하고, 각 행을 헤더명 key 의 dict 로
+    만든다. parser 의 _parse_tabular 와 동일 산출물(단일 소스). TabularCustomFieldsMapper.to_parse_format
+    입력 형태.
+    """
+    tables = load_tables(file_path, header_row=header_row, multi_table=multi_table)
+    data: list[dict] = []
+    for t in tables:
+        headers = t["headers"]
+        data_rows = [dict(zip(headers, values)) for values in t["data_rows"]]
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)